packages/core/src/App/Containers/RealAccountSignup/account-wizard.jsx

```diff
@@ -60,6 +63,28 @@ const AccountWizard = props => {
     const [should_accept_financial_risk, setShouldAcceptFinancialRisk] = React.useState(false);
     const is_high_risk_client_for_mt5 = useIsClientHighRiskForMT5();
 
+    const analytic_data = React.useMemo(
+        () => ({
+            form_source: document.referrer,
+            form_name: 'real_account_signup_form',
+            landing_company: props.real_account_signup_target,
+        }),
+        [props.real_account_signup_target]
+    );
+
+    const trackEvent = React.useCallback(
```

Comment:
do we need to cleanup anything in useEffect like you did in `personal-details.jsx`?

Reply:
That cleanup is for closing the identity step. That event is specific for personal details step. There's not close button there so added to unmount to trigger trackEvent